packages/create-next-app/templates/default-tw/ts/pages/_document.tsx

```diff
@@ -4,7 +4,7 @@ export default function Document() {
   return (
     <Html lang="en">
       <Head />
-      <body>
+      <body className="antialiased">
```

Comment:
This class name wasn't added to the above stylesheets, was this leftover?

Reply:
for the tailwind-based templates i'm using their classname `antialiased`

Reply:
it's built-in within the framework

Reply:
x-ref: https://tailwindcss.com/docs/font-smoothing#applying-font-smoothing